crhelper/utils.py

```diff
@@ -22,9 +26,18 @@ def _send_response(response_url, response_body):
     split_url = urlsplit(response_url)
     host = split_url.netloc
     url = urlunsplit(("", "", *split_url[2:]))
+    ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
+    if isinstance(ssl_verify, str) and path.exists(ssl_verify):
+        ctx.load_verify_locations(cafile=ssl_verify)
+    else:
+        logger.warning("Cert path {0} does not exist!.  Falling back to using system cafile.".format(ssl_verify))
```

Comment:
this fires if `ssl_verify=False` was the intent to have this warning only when `ssl_verify=path` and path does not exist?

Reply:
I've updated this logic in the latest commit